unittests/linuxliketests.py

```diff
@@ -533,6 +536,8 @@ def _test_stds_impl(self, testdir: str, compiler: 'Compiler') -> None:
                 continue
             elif '++20' in v and not has_cpp20:
                 continue
+            elif ('++23' in v or '++2b' in v) and not has_cpp23:
```

Comment:
Does it make sense to get the ++26 and ++2c in here as well? I see you added some support for that in another commit.

Reply:
I'd like to, but I am not sure when GCC will add support for the C++26 flags (could be next release like with Clang, but not certain), so I'd rather not just make a wild guess about the compatible compiler versions.

Reply:
GCC trunk does now, so 14 will: https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=5388a43f6a3f348929292998bd6d0c1da6f006de.

Reply:
Thanks, updated the commits.